manifests/base/cuda-rstudio-imagestream.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-creator: RHOAI
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
+    opendatahub.io/notebook-image-name: "RStudio | Minimal | CUDA | R 4.5"
+    opendatahub.io/notebook-image-desc: "RStudio Server Workbench image with an integrated development environment for R, a programming language designed for statistical computing and graphics."
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
```

Comment:
We should add also this annotation: 
`opendatahub.io/notebook-image-order: "22"`
But need to check what is the proper number for downstream.

(The same for the rstudio-imagestream.yaml)